test/inspect.js

```diff
@@ -2,6 +2,11 @@ var test = require('tape');
 var hasSymbols = require('has-symbols/shams')();
 var utilInspect = require('../util.inspect');
 var repeat = require('string.prototype.repeat');
+var URL;
+try {
+    // eslint-disable-next-line global-require
+    URL = require('url').URL;
+} catch (err) {}
```

Comment:
This will load an installed `url` module if it happens to be there; let's use `is-core-module` and conditionally require (without a try/catch)

Reply:
Actually, it should be fine by just using your other suggestion of having `typeof URL !== 'undefined'` in the test, as this will run the tests only if URL is in the global scope.